RecordParser/Engines/Reader/QuoteField.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+
+namespace RecordParser.Engines.Reader
+{
+    public static class QuoteField
+    {
+        public static (int start, int length) ParseQuotedChuck(in ReadOnlySpan<char> line, ref int scanned, ref int position, in ReadOnlySpan<char> delimiter)
+        {
+            const string singleQuote = "\"";
+
+            var unlook = line.Slice(scanned);
+
+            scanned += unlook.IndexOf(singleQuote) + 1;
+            unlook = line.Slice(scanned);
+            position = 0;
+
+            while (true)
+            {
+                position += unlook.Slice(position).IndexOf(singleQuote);
+                position++;
+                if (unlook.Slice(position).IndexOf(singleQuote) == 0)
```

Comment:
I really dont want to spent time discovering the indexOf, just wanna if it is zero
```suggestion
                if (unlook.Slice(position) is var temp && temp.IsEmpty == false && temp[0] == singleQuote)
```